main.go

```diff
@@ -256,7 +252,7 @@ func main() {
 		middleware.StripSlashes,
 		middleware.Recoverer,
 		middlewares.NewPrincipalMiddleware(),
-		middlewares.NewLoggingMiddleware(logbuch.Info, []string{
+		middlewares.NewLoggingMiddleware(slog.Info, []string{
```

Comment:
Could you replace [this part](https://github.com/muety/wakapi/blob/master/middlewares/logging.go#L45-L54) of the request logging middleware with structured logging as well, please? 

If we want to keep the middleware "generic" (accept arbitrary log functions), we'd probably need to pass it another parameter like `isStructured` or sth. and then have an `if-else` to log in either plain text (like now) or structured. But I'd also be fine with having the middleware specify its own logger instead of being passed a func. Whatever you think is cleaner.

Reply:
I missed this part, let me update it. We will use a structured logger anyway, and other implementations like Zap also follow the (key: value) approach. In case we change our mind in the future, this log func still works 🥇